ngraph/.clang-format

```diff
@@ -1,54 +1,31 @@
-BasedOnStyle: LLVM
+BasedOnStyle: Google
 IndentWidth: 4
 UseTab: Never
+
 Language: Cpp
 Standard: Cpp11
 
 AccessModifierOffset: -4
 
-AlignConsecutiveDeclarations: false
-AlignConsecutiveAssignments: false
-AlignTrailingComments: true
-
-AllowShortBlocksOnASingleLine: true
-AllowShortCaseLabelsOnASingleLine: true
-AllowShortFunctionsOnASingleLine: Inline
+AlignConsecutiveMacros: true
+AllowAllArgumentsOnNextLine: false
+AllowAllParametersOfDeclarationOnNextLine: false
 
-AlwaysBreakBeforeMultilineStrings: true
-AlwaysBreakTemplateDeclarations: true
+AllowShortFunctionsOnASingleLine: Empty
+AllowShortIfStatementsOnASingleLine: Never
+AllowShortLambdasOnASingleLine: Empty
+AllowShortLoopsOnASingleLine: false
 
-BinPackArguments: false
-BinPackParameters: false
+AlwaysBreakBeforeMultilineStrings: false
 
-BreakBeforeBraces: Allman
-BreakConstructorInitializersBeforeComma: true
+ColumnLimit: 160
```

Comment:
Can we go down to 120 here? A single file with so many columns is difficult to display on FullHD screen without horizontal scrolling(in an IDE with project tree open).

Reply:
Such limit is using in the IE. I propose to be aligned between components.

Reply:
So maybe other components could be changed as well.

Reply:
I will update the limit to 120 symbols

Reply:
The config was updated.